concepts/05 UI Components/DataGrid/20 Editing/45 Disable Editing.md

```diff
@@ -0,0 +1,46 @@
+DataGrid has API to disable editing a specific column, row and cell.
+
+### Column
```

Comment:
I suggest we make a "45 Disable Editing" folder and 4 .md files in it. 1 for intro, 2 for columns, 3 for rows and 4 for cells.

Reply:
ok